Simplenote/CoreDataManager.swift

```diff
@@ -35,24 +35,22 @@ class CoreDataManager: NSObject {
     private(set) var managedObjectContext: NSManagedObjectContext
     private(set) var persistentStoreCoordinator: NSPersistentStoreCoordinator
 
-    init(storageSettings: StorageSettings = StorageSettings()) throws {
+    init(at storageURL: URL, storageSettings: StorageSettings = StorageSettings()) throws {
```

Comment:
It should probably be up to the `StorageSettings` to return the correct `storageURL`